IsogeomGenerator/isg.py

```diff
@@ -364,6 +356,76 @@ def write_geometry(self, sname, sdir):
         self.mb.write_file(save_location)
         print("Geometry file written to {}.".format(save_location))
 
+    def __separate(self, ms):
+        """For a given surface meshset, separate meshset into unique and
+        disjoint surfaces based on their connectedness.
+
+        Input:
+        ------
+            ms: meshset entity handle
+
+        Returns:
+        --------
+            surf_list: list of entity handles for the set of unique and
+                disjoint surfaces in the meshset
+        """
+        surf_list = []
+
+        # get set of all vertices for the isosurface
+        remaining_verts = self.mb.get_entities_by_type(ms, types.MBVERTEX)
```

Comment:
It seems like this whole method is oriented towards set operations and might be better with everything as a set, by default, and only changed to a list when necessary? Right now, it's kind of a mixture and back-and-forth between them.

Reply:
I think that is what I do already- having it start as a list is necessary for lines 380 & 381. PyMOAB takes in a list and returns a list. So I think I am doing the bare minimum conversions already.